OpenImis.ePayment/Data/ImisBasePayment.cs

```diff
@@ -821,6 +821,117 @@ public void CancelPayment(int payment_id)
             }
         }
 
+        public object GetPaymentToReconciliate(object payment)
```

Comment:
Why object type and not an actual type?

Reply:
Ok. You are right. I wanted to use generic object. But using current Reconc type for sure might be more readable in code.  I change this.